api/server/v1/marshaler.go

```diff
@@ -1004,6 +1004,40 @@ func (x *ListInvoicesRequest) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// UnmarshalJSON on ListAppKeysRequest. Handles query param.
+func (x *ListAppKeysRequest) UnmarshalJSON(data []byte) error {
+	var mp map[string]jsoniter.RawMessage
+
+	if err := jsoniter.Unmarshal(data, &mp); err != nil {
+		return err
+	}
+
+	for key, value := range mp {
+		var v any
+
+		switch key {
+		case "key_type":
+			var keyType string
```

Comment:
just `v = &x.KeyType` for project as well.

Reply:
fixed it. Thanks